coupon-service/src/main/resources/application-batch-local.yml

```
@@ -0,0 +1,21 @@
+# 웹 서버를 아예 띄우지 않음
+server:
+  port: -1
+spring:
+  cloud:
+    discovery:
+      enabled: false
+  batch:
+    job:
+      enabled: false # 자동 job 실행 비활성화
```

YAMLlint (1.35.1)

[warning] too few spaces before comment

(comments)